content/en/docs/application-store/external-apps/deploy-clickhouse.md

```diff
@@ -14,6 +14,7 @@ This tutorial demonstrates how to deploy ClickHouse Operator and a ClickHouse Cl
 
 - You need to enable [the OpenPitrix system](../../../pluggable-components/app-store/).
 - You need to create a workspace, a project, and two user accounts (`ws-admin` and `project-regular`) for this tutorial. The account `ws-admin` must be granted the role of `workspace-admin` in the workspace, and the account `project-regular` must be invited to the project with the role of `operator`. This tutorial uses `demo-workspace` and `demo-project` for demonstration. If they are not ready, refer to [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/).
+- You need to set a gateway in KubeSphere for the external access to Services and Routes. For more information, see [Project Gateway](../../../project-administration/project-gateway/).
```

Comment:
gateway in KubeSphere
->
gateway in your KubeSphere cluster

Reply:
done